gatsby_fp/src/styles/facultyprofile.css

```diff
@@ -1,14 +1,23 @@
 /* Styles for Faculty Profile */
 
 body {
-  font-size: 1.1em;
+  font-family: "Chaparral W01", Georgia, Times, "Times New Roman", serif;
+  font-style: normal;
+  font-weight: 300;
+  min-width: 300px;
```

Comment:
On prod, there are some minor problems with small screens: the general info table gets horizontal scrolling when the viewport width is below approximately 418px, and content starts actually being obscured (requiring that the user horizontally scroll to reveal it) when the viewport width is below approximately 372px.

On this deploy preview, this is exacerbated by about 40-50 pixels: horizontal scrolling is activated below approximately 462px and content is actually obscured below approximately 419px.

40-50 pixels doesn't sound like a lot, but actually this brings the issue from "doesn't happen on many modern phones" to "happens on almost every phone." To easily see this, open prod and the deploy preview in separate tabs, set the Developer Tools in Chrome to the mobile emulator, and select something like "iPhone X." Also try selecting "Responsive" and drag the viewport width to anything reasonable.

Rather than send this back, I recommend we take the liberty of pulling the photo out of the table and float it such that it continues to appear to the left of the general info on sufficiently wide viewports, but on narrow viewports the photo appears above the general info. At that point there is no need for a table at all, as it would only be one column and therefore entirely useless.

While we're at it, we should put the job title between the person's name and their photo, regardless of viewport width. I suggested this on one of our calls and it was well-received, as it will greatly reduce the awkward negative space below the photo every time the photo is shorter than the info beside it.